programs/cleos/main.cpp

```diff
@@ -195,6 +195,10 @@ vector<string> tx_permission;
 
 eosio::client::http::http_context context;
 
+const string token_cntr_name = "cyber.token";
+const string wrap_cntr_name = "cyber.wrap";
+const string msig_cntr_name = "cyber.msig";
```

Comment:
it's better to set `name`s instead of `string`s: `const auto token_contract = N(cyber.token);`
`name` will be automatically converted to `string` in most cases where needed. you can use `.to_string()` in remaining cases